qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -387,7 +385,21 @@ def simplify(self, atol=None, rtol=None):
 
         # Pack bool vectors into np.uint8 vectors by np.packbits
         array = np.packbits(self.paulis.x, axis=1) * 256 + np.packbits(self.paulis.z, axis=1)
-        _, indexes, inverses = np.unique(array, return_index=True, return_inverse=True, axis=0)
+
+        # The following corresponds to
+        # _, indexes, inverses = np.unique(array, return_index=True, return_inverse=True, axis=0)
+        table = {}
+        indexes = []
+        inverses = np.zeros(array.shape[0], dtype=int)
```

Comment:
I think `np.empty` is faster than `np.zeros` as it doesn't initialize the array with zeros
```suggestion
        inverses = np.empty(array.shape[0], dtype=int)
```

Reply:
Thanks!